docs/configure/typescript.md

```diff
@@ -6,7 +6,7 @@ Storybook has built-in Typescript support, so your Typescript project should wor
 
 ### Default configuration
 
-The base Typescript configuration uses [`babel-loader`](https://webpack.js.org/loaders/babel-loader/) for Typescript transpilation, and optionally [`fork-ts-checker-webpack-plugin`](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/blob/v4.1.6/README.md#options) for checking.
+The base Typescript configuration uses [`babel-loader`](https://webpack.js.org/loaders/babel-loader/) for Typescript transpilation, and optionally <a href="https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/blob/v4.1.6/README.md#options"><code>fork-ts-checker-webpack-plugin</code></a> for checking.
```

Comment:
why not use markdown?

Reply:
@shilman to give a bit more context on this. Once the documentation gets pulled into the website we have a mechanism in place that adjusts the links for our documentation and generates the urls. It works for most cases, but for this particular item, it will lead to a 404 error, as the url that we get with markdown is (https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/blob/v4.1.6/README). I tried removing the readme and see if it work but it didn't. As we need to target that specific file in that specific version, we can prevent the 404 by adjusting it to a `<a>` and show the documentation for that package. 
Once version 5.x is implemented into Storybook we can revert this back to markdown.

Reply:
That's pretty bizarre! Thanks for clarifying 🙏